### DISCUSSION TITLE
Set analysis parameters hg19/hg38?

### DISCUSSION THREAD

**Discussion by yabeykoon** [Katsevich-Lab/sceptre#120] [Q&A] [answered]
Hi, 
In the 'set analysis parameters' page it says, "gene_position_data_frame_grch38 was derived from the GrCh38 (hg19) reference genome". Do you mean hg38? I just wanted to make sure they match up and which one is correct before performing my cis analysis. Thanks.

**Comment by timothy-barry** [comment, marked as answer]
Hi,

`gene_position_data_frame_grch38` is based on the GrCh38 (*not* hg19) reference genome. The book contained an error, which I just fixed. Thank you for pointing this out!
